wazo_test_helpers/asset_launching_test_case.py

```diff
@@ -346,25 +379,87 @@ def _maybe_dump_docker_logs(cls):
             logger.debug('Container logs dumped to %s', logfile.name)
 
     @staticmethod
-    def get_log_directory():
+    def get_log_directory() -> str:
         if not AssetLaunchingTestCase.log_dir:
-            char_set = string.ascii_lowercase
-            default_logging_dir = '/tmp/wazo-integration-{}'.format(
-                ''.join(random.choice(char_set) for _ in range(8))
+            random_strings = "".join(
+                random.choice(string.ascii_lowercase) for _ in range(8)
             )
             AssetLaunchingTestCase.log_dir = os.getenv(
-                'WAZO_TEST_DOCKER_LOGS_DIR', default_logging_dir
+                'WAZO_TEST_DOCKER_LOGS_DIR', f'/tmp/wazo-integration-{random_strings}'
             )
             if not os.path.exists(AssetLaunchingTestCase.log_dir):
                 os.makedirs(AssetLaunchingTestCase.log_dir, mode=0o755)
-        return AssetLaunchingTestCase.log_dir
+        return str(AssetLaunchingTestCase.log_dir)
 
 
-def _run_cmd(cmd, stderr=True):
-    logger.debug('%s', cmd)
-    stderr = subprocess.STDOUT if stderr else None
+class AssetLaunchingTestCase(AbstractAssetLaunchingHelper, unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.launch_service_with_asset()
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        cls.stop_service_with_asset()
+
+    @classmethod
+    def is_managing_containers(cls) -> bool:
+        warnings.warn(
+            "Method `is_managing_containers` was removed, please use "
+            "`get_container_management_enabled` or migrate to fixtures.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return get_container_management_enabled()
```

Comment:
Any reason to keep this compatibility? I didn't found any usage in the code
If something has to break, I want to be notified now :)
```suggestion
```

Reply:
I just figured it was a good practice, but if you prefer it just break existing code we can do it that way too :P